Makefile_packaging.mk

```diff
@@ -78,19 +78,19 @@ _topdir/SOURCES/%: % | _topdir/SOURCES/
 	ln $< $@
 
 $(NAME)-$(VERSION).tar.$(SRC_EXT).asc: $(NAME).spec Makefile
-	rm -f $(NAME)-$(VERSION).tar.{gz,bz*,xz}.asc
+	rm -f ./$(NAME)-*.tar.{gz,bz*,xz}.asc
 	curl -f -L -O '$(SOURCE).asc'
 
 $(NAME)-$(VERSION).tar.$(SRC_EXT): $(NAME).spec Makefile
-	rm -f $(NAME)-$(VERSION).tar.{gz,bz*,xz}
+	rm -f ./$(NAME)-*.tar.{gz,bz*,xz}
 	curl -f -L -O '$(SOURCE)'
 
 v$(VERSION).tar.$(SRC_EXT): $(NAME).spec Makefile
-	rm -f v$(VERSION).tar.{gz,bz*,xz}
+	rm -f ./v*.tar.{gz,bz*,xz}
 	curl -f -L -O '$(SOURCE)'
 
 $(VERSION).tar.$(SRC_EXT): $(NAME).spec Makefile
-	rm -f $(VERSION).tar.{gz,bz*,xz}
+	rm -f ./*.tar.{gz,bz*,xz}
```

Comment:
This one feels a little dangerous,

An alternate approach to just removing wildcard globs is that every build writes the name of it's tarball to a file so that a subsequent build can just remove the file named in that file.

So something like:
```
$(VERSION).tar.$(SRC_EXT): $(NAME).spec Makefile
	rm -f $(cat rm_me.txt)
	echo "$(notdir $(SOURCE))" > rm_me.txt
	curl -f -L -O '$(SOURCE)'
```

Then you know exactly what to remove from the last use of the workspace.

But not really saying to implement that if you don't want.  Just spitballing ideas while I'm thinking about it.

Reply:
Longer term, I think the tarball should be downloaded into its own directory that can be cleaned out if needed.